app/src/main/java/de/danoeh/antennapod/fragment/AddFeedFragment.java

```diff
@@ -50,38 +52,41 @@
     private static final int REQUEST_CODE_CHOOSE_OPML_IMPORT_PATH = 1;
     private static final int REQUEST_CODE_ADD_LOCAL_FOLDER = 2;
 
-    private EditText combinedFeedSearchBox;
     private MainActivity activity;
 
+    private AddfeedBinding viewBinding;
+    private EditTextDialogBinding alertViewBinding;
```

Comment:
I think this variable can be moved to the method that actually uses it. There is no need to have it accessible in the whole class.

Reply:
should I name both as viewBinding?

Reply:
I would use different names. Otherwise, it could be confusing when looking at the code and not realizing that there is a second variable with the same name.